01-cloudformation/README.md

```diff
@@ -227,13 +242,17 @@ Delete your CFN stacks in the same order you created them in. Did you
 succeed? If not, describe how you would _identify_ the problem, and
 resolve it yourself.
 
+#### I did not succeed. To diagnose, I went to the Cloudformation console and opened the events tab to find the failure. In this case, this stack cannot be deleted until the export dependency is removed from my second stack.
```

Comment:
Correct. You could also decouple the stacks by defining the resource name explicitly instead of importing stack 1's exported values. Importing causes a strict dependency while defining the resource name creates a soft-dependency.